docs/RestoreDatabase.md

```diff
@@ -0,0 +1,210 @@
+
+## Restore operations
+
+1. Scale in the Scalar product pods to **0** for stopping requests from Scalar products to backend databases.
+
+   You can scale in the Scalar product pods to 0 using the `--set *.replicaCount=0` flag of the helm command.
+
+   * ScalarDB Server
+     ```console
+     helm upgrade <release name> scalar-labs/scalardb -n <namespace> -f /path/to/<your custom values file for ScalarDB Server> --set scalardb.replicaCount=0
+     ```
+   * ScalarDL Ledger
+     ```console
+     helm upgrade <release name> scalar-labs/scalardl -n <namespace> -f /path/to/<your custom values file for ScalarDL Ledger> --set ledger.replicaCount=0
+     ```
+   * ScalarDL Auditor
+     ```console
+     helm upgrade <release name> scalar-labs/scalardl-audit -n <namespace> -f /path/to/<your custom values file for ScalarDL Auditor> --set auditor.replicaCount=0
+     ```
+
+1. Restore databases.
+
+   Restore the database data using the PITR feature. Please refer to the [Restore databases](./RestoreDatabase.md#restore-databases) section in this document.
+
+   If you use NoSQL or multiple databases, it is recommended to specify the mid-time of the **period** that you created in the document [Backup NoSQL database guide](./BackupNoSQL.md).
+
+1. Update database.properties/ledger.properties/auditor.properties based on the newly restored database.
+
+   The PITR feature restores databases as another instance. So, you must update endpoint information in the custom values file of Scalar products to access the newly restored databases.
+
+   Please refer to the following document for more details on how to configure the custom values file.
+
+   * [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md)
+
+   Note that if you use Amazon DynamoDB, it restores data with another table name instead of another instance. In other words, the endpoint is not changed by restoring data. Instead, you need to restore and rename tables in Amazon DynamoDB. Please refer to the [Amazon DynamoDB](./RestoreDatabase.md#amazon-dynamodb) section in this document.
+
+1. Scale out the Scalar product pods to more than 1 to start accepting requests from clients.
+
+   You can scale out the Scalar product pods using the `--set *.replicaCount=N` flag of the helm command.
+
+   * ScalarDB Server
+     ```console
+     helm upgrade <release name> scalar-labs/scalardb -n <namespace> -f /path/to/<your custom values file for ScalarDB Server> --set scalardb.replicaCount=3
+     ```
+   * ScalarDL Ledger
+     ```console
+     helm upgrade <release name> scalar-labs/scalardl -n <namespace> -f /path/to/<your custom values file for ScalarDL Ledger> --set ledger.replicaCount=3
+     ```
+   * ScalarDL Auditor
+     ```console
+     helm upgrade <release name> scalar-labs/scalardl-audit -n <namespace> -f /path/to/<your custom values file for ScalarDL Auditor> --set auditor.replicaCount=3
+     ```
```

Comment:
```suggestion
1. Scale in ScalarDB or ScalarDL pods to **0** to stop requests to the backend databases. You can scale in the pods to **0** by using the `--set *.replicaCount=0` flag in the helm command.
   * ScalarDB Server
     ```console
     helm upgrade <release name> scalar-labs/scalardb -n <namespace> -f /path/to/<your custom values file for ScalarDB Server> --set scalardb.replicaCount=0
     ```
   * ScalarDL Ledger
     ```console
     helm upgrade <release name> scalar-labs/scalardl -n <namespace> -f /path/to/<your custom values file for ScalarDL Ledger> --set ledger.replicaCount=0
     ```
   * ScalarDL Auditor
     ```console
     helm upgrade <release name> scalar-labs/scalardl-audit -n <namespace> -f /path/to/<your custom values file for ScalarDL Auditor> --set auditor.replicaCount=0
     ```
2. Restore the databases by using the point-in-time recovery (PITR) feature. For details on how to restore the databases based on your managed database, please refer to the [Supplemental procedures to restore databases based on managed database](./RestoreDatabase.md#supplemental-procedures-to-restore-databases-based-on-managed-database) section in this guide.

   If you are using NoSQL or multiple databases, you should specify the middle point of the period that you created when following the backup procedure in [Back up a NoSQL database in a Kubernetes environment](./BackupNoSQL.md).
3. Update **database.properties**, **ledger.properties**, or **auditor.properties** based on the newly restored database. Because the PITR feature restores databases as another instance, you must update the endpoint information in the custom values file of ScalarDB or ScalarDL to access the newly restored databases. For details on how to configure the custom values file, see [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md).

   Please note that, if you are using Amazon DynamoDB, your data will be restored with another table name instead of another instance. In other words, the endpoint will not change after restoring the data. Instead, you will need to restore the data by renaming the tables in Amazon DynamoDB. For details on how to restore data with the same table name, please see the [Amazon DynamoDB](./RestoreDatabase.md#amazon-dynamodb) section in this guide.
4. Scale out the ScalarDB or ScalarDL pods to **1** or more to start accepting requests from clients by using the `--set *.replicaCount=N` flag in the helm command.
   * ScalarDB Server
     ```console
     helm upgrade <release name> scalar-labs/scalardb -n <namespace> -f /path/to/<your custom values file for ScalarDB Server> --set scalardb.replicaCount=3
     ```
   * ScalarDL Ledger
     ```console
     helm upgrade <release name> scalar-labs/scalardl -n <namespace> -f /path/to/<your custom values file for ScalarDL Ledger> --set ledger.replicaCount=3
     ```
   * ScalarDL Auditor
     ```console
     helm upgrade <release name> scalar-labs/scalardl-audit -n <namespace> -f /path/to/<your custom values file for ScalarDL Auditor> --set auditor.replicaCount=3
     ```
```